src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -95,6 +95,8 @@ class Request
 
     /**
      * Query string parameters ($_GET).
+     *
+     * @var InputBag<string>
```

Comment:
Cannot `InputBag` contain other types than `string`?

Reply:
values in the query string (and cookie) are always strings
(the payload can contain something else, when decoding jsons)